src/components/SettingsDialog/ExportBoard/ExportHintHiddenColumns/ExportHintHiddenColumns.scss

```diff
@@ -0,0 +1,48 @@
+@import "src/constants/style.scss";
+
+$icon-text--gap: 20px;
+$columns-list--padding-left: $icon-text--gap + 50px;
+$list-style-type: "– ";
```

Comment:
Why are you defining a variable for single use?

Reply:
I consider the $list-style-type more as a constant, which I like to put on the top so you can see it right away:)